src/core/fieldformatter/qgsvaluerelationfieldformatter.h

```diff
@@ -39,16 +39,18 @@ class CORE_EXPORT QgsValueRelationFieldFormatter : public QgsFieldFormatter
     struct ValueRelationItem
     {
       //! Constructor for ValueRelationItem
-      ValueRelationItem( const QVariant &key, const QString &value )
+      ValueRelationItem( const QVariant &key, const QString &value, const QString &description )
```

Comment:
I believe this is an API break
```suggestion
      ValueRelationItem( const QVariant &key, const QString &value, const QString &description = QString() )
```